mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Annotation.test

```diff
@@ -0,0 +1,39 @@
+################################################################
+# Licensed to the Apache Software Foundation (ASF) under one   #
+# or more contributor license agreements.  See the NOTICE file #
+# distributed with this work for additional information        #
+# regarding copyright ownership.  The ASF licenses this file   #
+# to you under the Apache License, Version 2.0 (the            #
+# "License"); you may not use this file except in compliance   #
+# with the License.  You may obtain a copy of the License at   #
+#                                                              #
+#   http://www.apache.org/licenses/LICENSE-2.0                 #
+#                                                              #
+# Unless required by applicable law or agreed to in writing,   #
+# software distributed under the License is distributed on an  #
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       #
+# KIND, either express or implied.  See the License for the    #
+# specific language governing permissions and limitations      #
+# under the License.                                           #
+################################################################
+C: a0 CAPABILITY
+S: \* CAPABILITY .* ANNOTATION .*
+S: a0 OK CAPABILITY completed.
+
+C: a1 SETMETADATA INBOX (/private/comment "My new comment")
```

Comment:
I think that's enought for set (invalid case, valid case, multi entries, multi lines, NIL) but depend on the getMetadata, we need to add more.
The other specal cases were tested by UT. WDYT?